cmd/scheduled-feed/main.go

```diff
@@ -97,8 +98,25 @@ func main() {
 		pub:       pub,
 		delta:     delta,
 	}
+
+	if appConfig.Timer {
+		cronjob := cron.New()
+		crontab := fmt.Sprintf("@every %s", delta.String())
+		log.Printf("Running a timer %s", crontab)
+		cronjob.AddFunc(crontab, func() { cronrequest(appConfig.HttpPort) })
+		cronjob.Start()
+	}
+
 	http.Handle("/", handler)
 	if err := http.ListenAndServe(fmt.Sprintf(":%v", appConfig.HttpPort), nil); err != nil {
 		log.Fatal(err)
 	}
 }
+
+func cronrequest(port int) {
+	resp, err := http.Get(fmt.Sprintf("http://localhost:%v", port))
```

Comment:
It would good to have timeout on HTTP get.

Reply:
Added a timeout inline with the codebase